### DISCUSSION TITLE
NGE

### DISCUSSION THREAD

**Discussion by wowakan** [pwillworth/galaxyharvester#50] [General]
Hello,
I know this works with Pre-CU :). Will this work with NGE at all?

Kind Regards,
Wowakan

**Comment by pwillworth** [comment]
Yeah, sure.  There is a box you check when adding a new galaxy to say if it is NGE.  Different schematic sets are displayed when you have an NGE galaxy selected.

**Comment by wowakan** [comment]
With being able to do NGE now do we have to add it to the server or do can we put it in a website, say on Plesk?

I wanted to add on my web server(Not the game server) I use litespeed.
